opm/simulators/utils/readDeck.cpp

```diff
@@ -439,15 +452,21 @@ Opm::setupLogging(const int          mpi_rank_,
             : std::filesystem::current_path().generic_string();
     }
 
+    cleanDeck(baseName, output_dir);
```

Comment:
should this run on all processes? of course shouldn't harm but also a bit of a eyebrow-raiser.

Reply:
Yes, I was thinking about this as well.
I saw that the check if the output directory output_dir exists (and creating it if it does not exist) is also done on all processes though... imho things like this should be consistent. What do you think?

Reply:
i agree on consistency. this is okay then, we can consider addressing both afterwards.

Reply:
Regardless of what other mistakes we've made in this area, this particular function (`cleanDeck()`, although I might suggest `removePreviousResultFiles()` or `prepareResultOutputDirectory()` instead) should only run on the I/O rank and other ranks should wait for its completion.